BUILD.bazel

```diff
@@ -44,6 +44,11 @@ filegroup(
         "//examples:all-srcs",
         "//federation:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
```

Comment:
Hm the `hack/update-bazel.sh` script should not have added these lines. Is your git tree checked out into a directory structure that ends in `*/src/k8s.io/kubernetes`? For now I think you can just revert this file and the PR should pass, but for the future you should figure out why the script did this in your environment.